apps/omg_eth/test/omg_eth/subscription_worker_test.exs

```diff
@@ -25,7 +25,7 @@ defmodule OMG.Eth.SubscriptionWorkerTest do
   @moduletag :common
 
   setup do
-    _ = Agent.start_link(fn -> 55_555 end, name: :port_holder)
+    _ = Agent.start_link(fn -> 55_556 end, name: :subscription_port_holder)
```

Comment:
It might be better if you go into a different range. Others take 55_555 as the starting port and gradually increment :)